torch/csrc/jit/passes/quantization.cpp

```diff
@@ -572,5 +600,80 @@ template TORCH_API void InsertQuantDequantNodesForParam(
         getQParamFunc,
     at::ScalarType t);
 
+
+TORCH_API script::Module InsertObservers(
```

Comment:
Do you eventually plan to remove the older InsertObserver method because i believe you would mostly be operating at Module level. I see there is some code duplication

Reply:
yes I will do that in separate PRs